internal/Picker/Picker.js

```diff
@@ -332,11 +403,21 @@ const PickerBase = kind({
 						aria-label={decrementAriaLabel}
 						className={css.secondaryItemDecrement}
 						disabled={isSecond}
-						onClick={handleSecondaryDecrement}
+						onClick={() => {
+							handleDecrement(); setTimeout(() => handleDecrement(), 150);
```

Comment:
Is there any reason that we have to use `setTimeout` here?

Reply:
Yes. If we don't set a timeout the 2 functions are called simultaneously and we can't see the double transition/animation.